From 88085caf87cf1060c0db269f021efdea617fbef9 Mon Sep 17 00:00:00 2001
From: Tomohiro Fujiwara <tomohiro.fujiwara.cw@hitachi.com>
Date: Wed, 26 Sep 2018 10:12:02 +0900
Subject: [PATCH 1/2] [OPTEE_PROVIDER][#188185] Fix a contxt size allocated by
 the OP-TEE OS

This commit adds a compile option for deciding a context size used by
HASH algorithm of SS6.3-Secure Driver.

Signed-off-by: Tomohiro Fujiwara <tomohiro.fujiwara.cw@hitachi.com>
---
 core/core.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/core.mk b/core/core.mk
index c428cd00..405b1e76 100644
--- a/core/core.mk
+++ b/core/core.mk
@@ -90,6 +90,7 @@ base-prefix :=
 
 ifeq ($(CFG_CRYPT_HW_CRYPTOENGINE),y)
 core-platform-cflags += -DENABLE_CRYPTOENGINE
+core-platform-cflags += -DDX_CC_TEE -DCRYS_NO_CRYS_COMBINED_SUPPORT
 
 ifeq ($(CFG_CRYPT_ENABLE_CEPKA),y)
 libname = crypto_engine_pka
-- 
2.14.1.windows.1

